core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.html

```diff
@@ -24,29 +28,42 @@ <h4 class="col oppia-translation-modal-author-header">
       <div class="oppia-suggestion-review-panel-header-container">
         <strong>Content to translate:</strong>
       </div>
-      <div class="oppia-suggestion-review-panel">
+      <div class="oppia-suggestion-review-panel"
+           [ngClass]="isContentExpanded ? 'oppia-expanded' : 'oppia-contracted'"
+           #contentContainer>
```

Comment:
Why use ID here?

Reply:
This is needed for selected this div in angular using @ViewChild (for calculating panel height).